Mrittika-Chittagong.md

```diff
@@ -0,0 +1,8 @@
+## Chittagon, Bangladesh
+
+### Fun Facts 
+
```

Comment:
Chittagong is the port city so it is said that the whole country moves with us(Chittagong).